tools/tests/test_extensions.py

```diff
@@ -46,7 +46,7 @@ def test_2_daemon_api(self):
 
         # Get a python-based thrift client
         client = test_base.EXClient(daemon.options["extensions_socket"])
-        self.assertTrue(client.open(timeout=EXTENSION_TIMEOUT))
+        self.assertTrue(client.open())
```

Comment:
Why removing this timeout?
It should be the timeout for the extension to register with the core.

Reply:
`EXClient` in `test_base.py` did not support this option anymore: ` def open(self, attempts=10, interval=0.5):`.

Reply:
Ah yeah it still has the same meaning but it wasn't being passed to osquery.
All good!

Reply:
:+1: